src/main/java/mate/academy/springboot/datajpa/dto/request/CategoryRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.springboot.datajpa.dto.request;
+
+import lombok.Data;
+
+@Data
```

Comment:
It's bad practice to use `@Data`, better to use single annotations you need

Reply:
It’s a good rule for entities. for Dto we can use this annotation